presto-cassandra/src/test/resources/cu-cassandra.yaml

```diff
@@ -525,10 +525,10 @@ request_scheduler: org.apache.cassandra.scheduler.NoScheduler
 # the request scheduling. Currently the only valid option is keyspace.
 # request_scheduler_id: keyspace
 
-# index_interval controls the sampling of entries from the primrary
+# index_interval controls the sampling of entries from the primary
 # row index in terms of space versus time.  The larger the interval,
-# the smaller and less effective the sampling will be.  In technicial
-# terms, the interval coresponds to the number of index entries that
+# the smaller and less effective the sampling will be.  In technician
+# terms, the interval corresponds to the number of index entries that
```

Comment:
Oops. This should be `technical`. To be fair to Google Sheets (which picked the correction), the edit distance is the same...